src/sentry/integrations/vercel/integration.py

```diff
@@ -68,14 +69,93 @@ def get_organization_config(self):
                 "type": "project_mapper",
                 "mappedDropdown": {
                     "items": vercel_projects,
-                    "placeholder": "Select a Vercel Project",  # TOOD: add translation
+                    "placeholder": "Select a Vercel Project",  # TODO: add translation
                 },
                 "sentryProjects": sentry_projects,
             }
         ]
 
         return fields
 
+    def update_organization_config(self, data):
+        # data = {"project_mappings": [[sentry_project_id, vercel_project_id]]}
+
+        metadata = self.model.metadata
+        vercel_client = VercelClient(metadata["access_token"], metadata.get("team_id"))
+        config = self.org_integration.config
+        sentry_project_id = data["project_mappings"][-1][
+            0
+        ]  # TODO: update this to work in the case where a project is removed
+        vercel_project_id = data["project_mappings"][-1][1]
+        sentry_project = Project.objects.get(id=sentry_project_id)
+        sentry_project_dsns = ProjectKey.objects.filter(project=sentry_project).all()
```

Comment:
@ceorourke You should include `is_active` in the actual filter conditions instead of doing it later.

Reply:
@scefali `is_active` is a property on `ProjectKey` so I can't filter on it, but I can mush it together like:

`enabled_dsns = [dsn for dsn in ProjectKey.objects.filter(project=sentry_project).all() if dsn.is_active]`

Reply:
@ceorourke Why can't you just do: `ProjectKey.objects.filter(project=sentry_project, status=ProjectKeyStatus.ACTIVE)`?

But as it turns out, there is a method on the `ProjectKey` class called `get_default` that might work as well. I am guessing this is what's used to show the default key in the UI.